test/sharness/README.md

```diff
@@ -1,4 +1,4 @@
-# ipfs whole tests using the [sharness framework](https://github.com/mlafeldt/sharness/)
+# ipfs whole tests using the [sharness framework](https://github.com/pl-strflt/sharness/tree/feat/junit)
```

Comment:
We were using https://github.com/mlafeldt/sharness/ with a git.patch before. I think a repo is better for visibility. I'll try to contribute `feat/junit` back to https://github.com/chriscool/sharness.